ownership/sync-repos.py

```diff
@@ -0,0 +1,255 @@
+"""
+Synchronize ownership information between repositories and spreadsheet.
+
+Finds mismatches between the repositories listed in the ownership
+tracking spreadsheet and the actual list of repositories in a
+collection of Github organizations.
+"""
+
+import click
+import csv
+import github3
+from github3.exceptions import NotFoundError
+from itertools import groupby
+import re
+
+def bail(message):
+    raise KnownError(message)
+
+class KnownError(Exception):
+    def __init__(self, message):
+        super().__init__(message)
+        self.message = message
+
+@click.command()
+@click.option(
```

Comment:
When we insert this as a jenkins job step, we'll want to be able to pass in the other settings as environment variables.  can you set it so that all these options can also be passed in via the environment: https://click.palletsprojects.com/en/7.x/options/#values-from-environment-variables